fusetrace/fusetrace_ll.cc

```diff
@@ -11,7 +11,7 @@
     gcc -Wall `pkg-config fuse --cflags --libs` -lulockmgr fusexmp_fh.c -o fusexmp_fh
 */
 
-#define FUSE_USE_VERSION 30
+#define FUSE_USE_VERSION 35
```

Comment:
I wish this could be defined on the compiler command-line or config.h instead of so many files.

Reply:
Good idea. Putting it in config.h was a bit tricky but we can at least move it to a common header file and just reorganize the includes a little to make sure it does the right thing.